azure-local/update/import-discover-updates-offline-23h2.md

```diff
@@ -5,16 +5,14 @@ author: ronmiab
 ms.author: robess
 ms.topic: how-to
 ms.reviewer: arduppal
-ms.date: 11/18/2025
+ms.date: 12/09/2025
 ---
 
 # Import and discover update packages with limited connectivity
 
 ::: moniker range=">=azloc-24113"
 
-[!INCLUDE [azure-local-banner-23h2](../includes/azure-local-banner-23h2.md)]
-
-This article explains how to discover and import solution update packages for Azure Local deployed in sites with limited bandwidth connections to Azure. Starting with version 2503, you can download Azure Local solution updates as static payloads. Download the payload once, copy or transfer it to multiple instances, and import it using PowerShell. Do these actions before you start an update to reduce the amount of data downloaded during the update.
+This article explains how to discover and import solution update packages for Azure Local (*formerly Azure Stack HCI*) deployed in sites with limited bandwidth connections to Azure. Starting with version 2503, you can download Azure Local solution updates as static payloads. Download the payload once, copy or transfer it to multiple instances, and import it using PowerShell. Do these actions before you start an update to reduce the amount of data downloaded during the update.
```

Comment:
Should we remove "Starting with version 2503" given that 2503 is officially out of support (as it is older than the last 6 releases)?